src/actions/plugins/surround.ts

```diff
@@ -485,6 +485,7 @@ export class CommandSurroundAddToReplacement extends BaseCommand {
         type: 'insertText',
         text: startReplace,
         position: start,
+        diff: new PositionDiff({ character: -startReplace.length }),
```

Comment:
For posterity's sake, could you please add a comment explaining what this `PositionDiff` is for?